src/lib/api/posts/posts.query-builder.ts

```diff
@@ -0,0 +1,28 @@
+import { PostsGetManyQueryParams } from "@/lib/api/posts/posts.types";
+
+export function buildPostsQueryString(
```

Comment:
i think we try to use arrow functions as lot as we can

export const buildPostsQueryString = () =>.....
instead of 
export function....